spec/mongo/operation/write/command/insert_spec.rb

```diff
@@ -56,6 +56,31 @@
     end
   end
 
+  describe 'write concern' do
+
+    context 'when write concern is not specified' do
+
+      let(:spec) do
+        { :documents     => documents,
+          :db_name       => db_name,
+          :coll_name     => coll_name,
+          :ordered       => true
+        }
+      end
+
+      it 'does not include write concern in the selector' do
+        expect(op.send(:selector)[:writeConcern]).to be_nil
```

Comment:
I think we're just not supposed to send write concern if the user doesn't specify it (and we're using write commands). If we don't send write concern, the server's default is used. Bernie verified this the other day:

emily
For write commands, we shouldn't send write concern unless the user explicitly sets one (at whatever level we allow it - client/db/collection)
Bernie
Yep

I'll double check with Craig though.